File: 0462-minimum-moves-to-equal-array-elements-ii/0462-minimum-moves-to-equal-array-elements-ii.java
class Solution {
    public int minMoves2(int[] nums) {
        Arrays.sort(nums);
       int mid=nums.length/2;
        int mov=0;
        for(int i=0;i<nums.length;i++){
             mov+=Math.abs(nums[i]-nums[mid]);
        }
       return mov;
    }
}

File: 0462-minimum-moves-to-equal-array-elements-ii/0462-minimum-moves-to-equal-array-elements-ii.cpp
class Solution {
public:
    int minMoves2(vector<int>& nums) {
        int n = nums.size(), diff = 0;
        sort(nums.begin(), nums.end());
        int mid = nums[n/2];
        for(int i=0; i<n; i++)  diff += abs(nums[i] - mid); 
        return diff;
    }
};